scripts/ci/pre_commit/pre_commit_breeze_cmd_line.py

```diff
@@ -79,6 +80,19 @@ def verify_all_commands_described_in_docs():
         sys.exit(1)
 
 
+def verify_breeze_installed():
+    result = subprocess.getoutput('breeze version')
```

Comment:
Maybe use `shutil.which()`? Not as accurate (fails if `breeze` is not “our” command but from somewhere else, for example), but a lot faster.

Reply:
Yeah. Shutil.which is better. It's also portable and it's enough (we don't verify the output of `version` command anyway.